docs/source/Internals.md

```diff
@@ -306,7 +306,7 @@ that may further help clarify the format.
 
 # FlexBuffers
 
-The [schema-less](@ref flexbuffers) version of FlatBuffers have their
+The [FlexBuffers](@ref flexbuffers) version of FlatBuffers have their
```

Comment:
You'd have to change the rest of the sentence as well if you want to make this change. FlexBuffers are not a "version of FlatBuffers", so maybe just "FlexBuffers (a schema-less serialization format similar to FlatBuffers) have their own encoding, detailed here"

Reply:
It was a mistake. I will fix it.